### PR TITLE
refactor(templates/PULL_REQUEST_TEMPLATE): standardize

### PR DESCRIPTION
**Pull request by michael-ciniawsky**

### CODE REVIEW

File: templates/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+
+```
+commit message body...
+```
+
+### `Type`
+
+> ℹ️  What types of changes does your code introduce?
+
+> 👉 _Put an `x` in the boxes that apply and delete all others_
```
Comment (sapegin) [inline templates/.github/PULL_REQUEST_TEMPLATE.md]: I think checkboxes are unnecessary if we’re asking people to remove items.

File: templates/.github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+
+- [ ] Bug (:label: Patch)
+- [ ] Feature (:label: Minor)
+- [ ] Breaking Change (:label: Major)
+
+### `Issues`
+
+> ℹ️ What issue (if any) are closed by your PR?
+
+> 👉  _Replace `#1` with the error number that applies and remove the ``` ` ```_
```
Comment (sapegin) [inline templates/.github/PULL_REQUEST_TEMPLATE.md]: Issue number?

File: templates/.github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (michael-ciniawsky) [reply templates/.github/PULL_REQUEST_TEMPLATE.md]: What do you suggest instead :) ?

Comment (sapegin) [reply templates/.github/PULL_REQUEST_TEMPLATE.md]: Either put Xs or remove items ;-)

Comment (bebraw) [reply templates/.github/PULL_REQUEST_TEMPLATE.md]: I would drop the [ ] bit and have only a list. The developer can then remove the bits that aren't needed.

**Comment by shellscape** [comment]
Superseded by #95

**Pull request closed without merging**